### PR TITLE
Add housekeeping module

### PR DESCRIPTION
**Pull request by masa-orca**
##### SUMMARY

- Add housekeeping module
- Add tests for housekeeping module

##### ISSUE TYPE

- New Module Pull Request

##### COMPONENT NAME

zabbix_housekeeping
##### ADDITIONAL INFORMATION

- This module for Zabbix 5.2 and higher.
- Therefore, I tested sanity check and playbook on local enviroment.
- In now, This PR cannot pass checks because Some documents which belong to other modules cannot pass sanity checks. Please see [this issue](https://github.com/ansible-collections/news-for-maintainers/issues/9)

```paste below

```

### CODE REVIEW

File: plugins/modules/zabbix_housekeeping.py [hunk 1]
```diff
@@ -0,0 +1,395 @@
...
+
+description:
+   - This module allows you to modify Zabbix housekeeping setting.
+
+author:
+    - ONODERA Masaru(@masa-orca)
+
+requirements:
+    - "zabbix-api >= 0.5.4"
+
```
Comment (D3DeFi) [inline plugins/modules/zabbix_housekeeping.py:25]:
```suggestion
version_added: 1.6.0
```

**Review by D3DeFi — APPROVED** [review]
good job! :) Thank you for contributing this

**Comment by D3DeFi** [comment]
I will be merging #649 very shortly, please rebase your branch on top of latest `main` so sanity tests can pass

**Comment by masa-orca** [comment]
@D3DeFi Thank you for your kindness.
I rebase this branch. In addition, I can pass sanity-checls.

File: plugins/modules/zabbix_housekeeping.py (hunk 1 above)
Comment (masa-orca) [reply plugins/modules/zabbix_housekeeping.py:27]: Thanks! I add it.

**Comment by D3DeFi** [comment]
Thank you for the new module! :))

**Pull request merged**